lesson_2/Lesson_2.1.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>For site A I think it was 7%, should make sure our numbers/data match.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I used site B</p>